src/main.rs

```diff
@@ -64,13 +65,20 @@ fn play_music(path: &str, do_cap: bool) -> Result<&str, ExitStatus> {
     }
 }
 
-fn scan_complete(uid: &str) -> &'static str {
+fn scan_complete(uid: &str) -> String {
     match uid {
         "mom" => {
-            return "aaa.mp3";
+            return "scans/aaa.mp3".to_string();
         },
         _ => {
-            return "scan-complete-mom.mp3";
+            // get all files from scans folder
+            let files = std::fs::read_dir("scans").unwrap();
+            // return a random file that isn't aaa.mp3
+            let mut rng = rand::thread_rng();
+            let mut file_list: Vec<_> = files.map(|f| f.unwrap().path()).collect();
+            file_list.retain(|f| f.to_str().unwrap() != "scans/aaa.mp3");
```

Comment:
tbh, seems easier to have a separate folder where `aaa.mp3` lives

Reply:
that just removes one line and it's not a particularly complicated line, so I think the cleanliness of having all the scan files in the same folder outweighs that

Reply:
I dunno, it feels a little jank